src/Microsoft.Identity.Client/IPublicClientApplication.cs

```diff
@@ -42,6 +39,11 @@ namespace Microsoft.Identity.Client
     /// </summary>
     public partial interface IPublicClientApplication : IClientApplicationBase
     {
+        /// <summary>
+        /// 
```

Comment:
proposing the following comment:

Tells if the application can the system web browser, therefore getting single-sign-on with web applications. This is today the case on Xamarin.Android, Xamarin.iOS, and the system web view is used by default. See https://aka.ms/msal-net-uses-web-browser